docs/release-notes/release-notes-1.6.0-rc1.1.md

```diff
@@ -0,0 +1,925 @@
+A new vote with the id `treasury` is now available as of this release.  After
+upgrading, stakeholders may set their preferences through their wallet or Voting
+Service Provider's (VSP) website.
+
+The primary goal of this change is to fully decentralize treasury spending so
+that it is controlled by the stakeholders via stake voting.
+
+See the initial
+[Politeia proposal](https://proposals.decred.org/proposals/c96290a2478d0a1916284438ea2c59a1215fe768a87648d04d45f6b7ecb82c3f)
+for more details.
+
+### Aggregate Fee Block Template Transaction Selection (Child Pays For Parent)
+
+The transactions that are selected for inclusion in block templates that
+Proof-of-Work miners solve have been modified to prioritize the overall fees of
+the entire transaction ancestor graph.
+
+This is beneficial for both miners and end users as it:
+
+- Helps maximize miner profit by ensuring that unconfirmed transaction chains
+  with higher aggregate fees are given priority over others with lower aggregate
+  fees
+- Provides a mechanism for users to increase the priority of an unconfirmed
+  transaction by spending its outputs with another transaction that pays higher
+  fees
+
+This is commonly referred to as Child Pays For Parent (CPFP) as the spending
+("child") transaction is able to increase the priority of the spent ("parent")
+transaction.
+
+### HTTPS Seeding
+
+The initial bootstrap process that contacts seeders to discover other nodes to
+connect to now uses a REST-based API over HTTPS.
+
+This change will be imperceptible for most users, with the exception that it
+accelerates the process of finding suitable candidate nodes that support desired
+services, particularly in the case of recently-introduced services that have not
+yet achieved widespread adoption on the network.
+
+The following are some key benefits of HTTPS seeders over the previous DNS-based
+seeders:
+
+- Support for non-standard ports
+- Advertisement of supported service
+- Better scalability both in terms of network load and new features
+- Native support for TLS-secured communication channels
+- Native support for proxies which allows the use of anonymous overlay networks
+  such as Tor and I2P
+- No need for a large DNSSEC dependency surface
+- Uses better audited infrastructure
+- More secure
+- Increases flexibility
+
+### Signature Validation And Other Crypto Operation Optimizations
+
+The underlying crypto code has been reworked to significantly improve its
+execution speed and reduce the number of memory allocations.  While this has
+more benefits than enumerated here, probably the most important ones for most
+stakeholders are:
+
+- Improved vote times since blocks and transactions propagate more quickly
+  throughout the network
+- The initial sync process is around 15% faster
+
+### Proactive Signature Cache Eviction
+
+Signature cache entries that are nearly guaranteed to no longer be useful are
+now immediately and proactively evicted resulting in overall faster validation
```

Comment:
Any estimate how much faster? (this will make it more tangible for the reader)

Reply:
It varies significantly depending on the size of the block, how many of the transactions were seen, etc.  There really isn't a good number to give here.